core/client/pom.xml

```diff
@@ -256,7 +221,7 @@
               <relocations>
                 <relocation>
                   <pattern>org.apache.thrift</pattern>
-                  <shadedPattern>shaded.alluxio.org.apache.thrift</shadedPattern>
+                  <shadedPattern>${project.artifactId}.org.apache.thrift</shadedPattern>
```

Comment:
addressed ... I decided to go with using the `shading.prefix` property so that we can document what should be its relationship to the artifact ID